piku/commands/serial.py

```
@@ -2,13 +2,28 @@
 from serial.serialutil import SerialException
 from serial.tools import list_ports
 from serial.tools.miniterm import Miniterm
+import platform
+
 from piku.core import config
 
 
 def default():
     ports = list_ports.comports()
-    if not ports: return None
-    return ports[0].device
+    if not ports:
+        return None
+    if platform.system() == "Darwin":
+        # On macOS, we don't want to return the Bluetooth COM ports
+        for p in ports:
+            try:
+                idx = p.device.index("usbmodem")
+                if idx:
+                    return p.device
+            except:
```

I was looking into this catch all exception and discovered that it might actually be good to check for a usb device on every platform.  My computers don't have additional serial ports, but I imagine some people do and I think CircuitPython should always be a usb serial port.  For linux and windows it seems like we can check the hwid for "usb" or alternatively some fields are only set for usb devices:

```python
ports = [p for p in ports if 'usb' in hasattr(p, 'location')] # location field only set on a usb device
ports = [p for p in ports if 'usb' in p.hwid.lower()]
```

https://pyserial.readthedocs.io/en/latest/tools.html#serial.tools.list_ports.ListPortInfo